bin/main.dart

```diff
@@ -90,7 +96,13 @@ void main(List<String> args) {
     }
 
     final sortedFile = sort.sortImports(
-        file.readAsLinesSync(), packageName, emojis, exitOnChange, noComments);
+      file.readAsLinesSync(),
+      packageName,
+      emojis,
+      exitOnChange,
+      noComments,
+      filePath: filePath,
+    );
```

Comment:
Is this from that vscode settings file? If so feel free to keep the file.

Reply:
No, it's me – I almost always (when it makes sense) add the trailing comma because it makes the code prettier :)

Reply:
I'll remove the whole `.vscode` directory and add it to `.gitignore`, but then it'd be nice to include some info in `README.md` or `CONTRIBUTING.md` that this project uses lineLength = 80

Reply:
That would be great, thank you!